js-exercises/update-object/updateObject.js

```diff
@@ -0,0 +1,17 @@
+
+function updateObject(indexParam, replacementCharacter, inputArray) {
```

Comment:
indexParam can be spelt fully. The function name says it's updateObject but we're taking inputArray. It's contradicting.